Friendly-plans/app/src/main/java/database/repository/TaskTemplateRepository.java

```diff
@@ -45,6 +47,11 @@ public TaskTemplate get(Long id) {
                 .list();
     }
 
+    public List<PlanTaskTemplate> getPlansWithThisTask(Long taskId) {
+        PlanTaskTemplateDao targetDao = daoSession.getPlanTaskTemplateDao();
+        return targetDao._queryTaskTemplate_PlanTaskTemplates(taskId);
+    }
```

Comment:
I think it could be better to implement this method in PlanTemplateRepository, cause we are trying to get PlanTemplates by taskId :) Also we can use related model (PlanTaskTemplateDao) properties to query PlanTemplate model and get this list in a more convenient way, for example:
```
public List<PlanTemplate> getPlansByTaskId(Long taskId) {
        return daoSession.getPlanTemplateDao()
                .queryBuilder()
                .where(PlanTaskTemplateDao.Properties.TaskTemplateId.eq(taskId))
                .list();
    }
```